### PR TITLE
Feature: Strikethrough support.

### LINKED ISSUE #6412: Autoformat should support strikethrough
## 📝 Provide detailed reproduction steps (if any)

Currently autoformat does not support strikethrough. It should be possible to strikethrough text with `~~`.

## 📃 Other details

*   Browser: Any
*   OS: Any

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by niegowski**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Strikethrough support. Closes ckeditor/ckeditor5#6412.

### CODE REVIEW

File: docs/_snippets/features/autoformat.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ ClassicEditor
 				'bold',
 				'italic',
 				'code',
+				'strikethrough',
```
Comment (Reinmar) [inline docs/_snippets/features/autoformat.js]: I think you need to add the plugin.

File: tests/autoformat.js [hunk 2]
```diff
@@ -365,6 +367,15 @@ describe( 'Autoformat', () => {
 			expect( getData( model ) ).to.equal( '<paragraph><$text code="true">foobar</$text>[]</paragraph>' );
 		} );
 
+		it( 'should replace both "~~`" with strikethrough', () => {
```
Comment (Reinmar) [inline tests/autoformat.js]:
```suggestion
		it( 'should replace both "~~" with strikethrough', () => {
```

**Review by Reinmar — CHANGES_REQUESTED** [review]
As commented.

**Review by jodator — APPROVED** [review]
LGTM after introduced changes.

**Pull request merged**